### DISCUSSION TITLE
How can I downgrade dde-control-center to version 15.1.1?

### DISCUSSION THREAD

**Discussion by anig1scur** [linuxdeepin/developer-center#2963] [Q&A | 问答板块] [answered]
Thanks for your great work! It looks awesome

I prefer `dde-control-center 15.1.1`,so is there an easy way that I can do ?

**Comment by RlZkhere** [comment, marked as answer]
Download the file of the dde-control-center that you want to use or go to the pkg manager's cache directory and use Pacman -U $PACKAGE or apt install./$PACKAGE
